assignments/arrays.js

```diff
@@ -63,34 +64,55 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+console.log(`Car 33 is a ` +inventory[32].car_year + ' '+inventory[32].car_make+ ' ' +inventory[32].car_model );
+// console.log(inventory[32].car_year)
 
 
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
 let lastCar = 0;
-console.log();
+lastCar = inventory.pop();
```

Comment:
Very cool way to approach this problem. Another possible solution is to set lastCar equal to the last item in the inventory array and then console log lastCar. Eg:
```
let lastCar = inventory[inventory.length - 1];
console.log(lastCar);
```